docs/fundamentals/database-collection.txt

```diff
@@ -0,0 +1,176 @@
+
+.. code-block:: php
+   :emphasize-lines: 7
+   
+   'default' => 'mongodb',
+
+   'connections' => [
+
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'dsn' => 'mongodb://localhost:27017/animals',
+       ], ...
+   ]
+
+When you set a default database connection, {+odm-short+} uses that
+connection for operations, but you can specify multiple database connections
+in your ``config/database.php`` file.
+
+The following example shows how to specify multiple database connections
+(``mongodb`` and ``mongodb_alt``) to access the ``animals`` and
+``plants`` databases:
+
+.. code-block:: php
+   
+   'connections' => [
+
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'dsn' => 'mongodb://localhost:27017/animals',
+       ],
+
+       'mongodb_alt' => [
+           'driver' => 'mongodb',
+           'dsn' => 'mongodb://localhost:27017/plants',
+       ]
+
+   ], ...
+
+.. note::
+
+   The MongoDB PHP driver reuses the same connection when
+   you create two clients with the same connection string. There is no
+   overhead in using two connections for two distinct databases, so you
+   do not need to optimize your connections.
+
+If your application contains multiple database connections and you want
+to store your model in a database other than the default, override the
+``$connection`` property when creating a ``Model`` class.
+
+The following example shows how to override the ``$connection`` property
+on the ``Flower`` model class to use the ``mongodb_alt`` connection.
+This directs {+odm-short+} to store the model in the ``flowers``
+collection of the ``plants`` database, instead of in the default database:
+
+.. code-block:: php
+
+   class Flower extends Model
+   {
+       protected $connection = 'mongodb_alt';
+   }
+
+.. _laravel-access-coll:
+
+Access a Collection
+-------------------
+
+When you create model class that extends
+``MongoDB\Laravel\Eloquent\Model``, {+odm-short+} stores the model data
+in a collection with a name that is the snake case plural form of your
+model class name.
+
+For example, if you create a model class called ``Flower``,
+Laravel applies the model to the ``flowers`` collection in the database.
+
+.. tip::
+
+   To learn how to specify a different collection name when defining a model class, see the
+   :ref:`laravel-model-customize-collection-name` section of the Eloquent
+   Model Class guide.
+
+We generally recommend that you use the Eloquent ORM to access a collection
+implicitly for code readability and maintainability. The following
+example specifies a find operation by using the ``Flower`` class, so
+Laravel retrieves results from the ``flowers`` collection:
+
+.. code-block:: php
+
+   Flower::where('name', 'Water Lily')->get()
+
+If you are not able to accomplish your operation by using an Eloquent
+model, you can access a collection by using a facade to access the query
+builder class ``DB`` and calling the ``collection()`` method. The
```

Comment:
Suggestion:

I think it can be more concise to use the "DB facade" nomenclature which the [Laravel docs use](https://laravel.com/docs/11.x/queries#running-database-queries).

```suggestion
model, you can access the query builder by calling the ``collection()`` method on the ``DB`` facade. The
```